src/factors/DynPoint2D.jl

```diff
@@ -5,31 +5,31 @@
 $(TYPEDEF)
 """
 mutable struct DynPoint2VelocityPrior{T <: SamplableBelief} <: AbstractPrior
-  z::T
+  Z::T
   DynPoint2VelocityPrior{T}() where {T <: SamplableBelief} = new{T}()
   DynPoint2VelocityPrior{T}(z1::T) where {T <: SamplableBelief} = new{T}(z1)
 end
 DynPoint2VelocityPrior(z1::T) where {T <: SamplableBelief} = DynPoint2VelocityPrior{T}(z1)
 
-getSample(cfo::CalcFactor{<:DynPoint2VelocityPrior}) = rand(cfo.factor.z)
+getSample(cfo::CalcFactor{<:DynPoint2VelocityPrior}) = rand(cfo.factor.Z)
 
 
 """
 $(TYPEDEF)
 """
 mutable struct DynPoint2DynPoint2{T <: SamplableBelief} <: AbstractRelativeRoots
-  z::T
+  Z::T
   DynPoint2DynPoint2{T}() where {T <: SamplableBelief} = new{T}()
   DynPoint2DynPoint2(z1::T) where {T <: SamplableBelief} = new{T}(z1)
 end
 
-getSample(cfo::CalcFactor{<:DynPoint2DynPoint2}) = rand(cfo.factor.z)
+getSample(cfo::CalcFactor{<:DynPoint2DynPoint2}) = rand(cfo.factor.Z)
 
-function (cfo::CalcFactor{<:DynPoint2DynPoint2})(z, xi, xj)
+function (cfo::CalcFactor{<:DynPoint2DynPoint2})(Z, xi, xj)
```

Comment:
id suggest keeping the function calls lower case, since this represents a single event (not the stochastic process `Z`).  As in one sample `z ~ Z`.

Reply:
ok, I will update